bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
@ankit01ojha You should take a look [here](https://github.com/fedora-infra/bodhi/blob/develop/bodhi/client/bindings.py#L307)

Did you test your change ? Did the program print "wrong password, please try again" ?